app.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def handle_student_input
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp.capitalize
+    parent_permission = handle_permission
+    case parent_permission
+    when 'Y'
+      parent_permission = true
+    when 'N'
+      parent_permission = false
+    else
+      puts 'Invalid selection, please choose from Y or N'
+      parent_permission = handle_permission
+    end
+    student = Student.new(age, name, parent_permission: parent_permission, person_type: 'student')
+    @people << student
+    puts ['Person created succsefully', ' ']
+  end
+
+  def handle_permission
+    print 'Has Parent permission? [Y/N]: '
+    gets.chomp.capitalize
+  end
+
+  def handle_teacher_input
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp.capitalize
+    print 'Specialization: '
+    specialization = gets.chomp.capitalize
+    teacher = Teacher.new(age, name, specialization, person_type: 'teacher')
+    @people << teacher
+    puts ['Person created successfully', ' ']
+  end
```

Comment:
- [ ] I am not able to create a `student` or a `teacher`
<img width="1440" alt="Screen Shot 2023-03-10 at 12 22 17" src="https://user-images.githubusercontent.com/44745855/224291616-613f223c-844a-4d69-a49e-a2d22156d1d0.png">

Reply:
- [ ] Remember to remove all the examples used in previous project